Adding project for Site Recovery CSMv2 based specs.

NehaRaje opened the pull request:
Adding a new project for specs of Site Recovery Services based on the CSMv2 APIs.
API functionality has been added only for the enterprise to enterprise replication provider.
Basic framework for tests has been provided and test added.

NehaRaje committed:
First cut for ResourceManagement based SiteRecovery changes.

NehaRaje committed:
Adding left out files.

NehaRaje committed:
Modifications for models.

NehaRaje committed:
Checking in placeholder tests.

Not being added to the solution.

NehaRaje committed:
Failover related RP and PE files.

NehaRaje committed:
Project level changes for file additions.

neetus15 committed:
Adding DataContractUtils

hikewalr committed:
Fixing the file reference for Utilities.

neetus15 committed:
Failover test related changes.

neetus15 committed:
Cleaning up TFO test.

neetus15 committed:
Putting back ProtectionTests placeholder file.

NehaRaje committed:
Changing DLL Name.

hikewalr committed:
RecoveryPlan tests.

sriramvu committed:
Merge pull request #2 from AsrOneSdk/neha-master

Changing DLL Name.

sriramvu committed:
updated the generated files

sriramvu committed:
Merge branch 'master' of https://github.com/AsrOneSdk/azure-sdk-for-net

Conflicts:
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Generated/IJobOperations.cs
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Generated/IProtectionEntityOperations.cs
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Generated/JobOperationsExtensions.cs
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Generated/ProtectionContainerOperations.cs
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Generated/ProtectionEntityOperations.cs
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Generated/ProtectionProfileOperations.cs
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Generated/RecoveryPlanOperations.cs
	src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Microsoft.Azure.Management.RecoveryServicesARM.nuget.proj

neetus15 committed:
vm failover changes

NehaRaje committed:
Merge pull request #3 from AsrOneSdk/hikewalr-master

RecoveryPlan tests.

NehaRaje committed:
Adding generated files to csproj

NehaRaje committed:
Merge branch 'master' into neetus-master

NehaRaje committed:
Merge pull request #1 from AsrOneSdk/neetus-master

Adding failover tests.

NehaRaje committed:
Merge branch 'master' into neha-master

NehaRaje committed:
Checking in generated files.

Adding json session records for tests.

Cleaning up tests and resetting nuget package version.

NehaRaje committed:
Cleaning up tests and adding session records.

NehaRaje committed:
Resetting the nuget package version.

NehaRaje committed:
Merge pull request #4 from AsrOneSdk/neha-master

Checking in generated files and cleaning up tests.

NehaRaje committed:
Merge branch 'master' of https://github.com/Azure/azure-sdk-for-net

azuresdkci commented:
Can one of the admins verify this patch?

azurecla commented:
Hi __@NehaRaje__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@NehaRaje__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

stankovski commented:
@azuresdkci add to whitelist

stankovski commented on src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,22 @@
+
+using System.Reflection;
+
+[assembly: AssemblyTitle("Microsoft.Azure.Management.RecoveryServicesARM.Specification")]
```
Add AssemblyVersion and AssemblyFileVersion explicitly

NehaRaje committed:
Taking in PR comments.

Adding AssemblyVersion and AssemblyFileVersion.
Adding the tag to csproj to include the jsons.

NehaRaje committed:
Merge pull request #5 from AsrOneSdk/neha-master

Taking in PR comments.

hovsepm commented on src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Microsoft.Azure.Management.RecoveryServicesARM.nuget.proj:
```diff
@@ -0,0 +1,12 @@
+    <!--
+    Microsoft.Azure.Management.RecoveryServicesARM
+    -->
+    <SdkNuGetPackage Include="Microsoft.Azure.Management.RecoveryServicesARM">
```
please remove ARM suffix from the package name in the nuget.proj and nuspec files and their names.

sriramvu committed:
removed the ARM suffix and renamed to SiteRecovery
